### PR TITLE
Merging PR moves the card

### LINKED ISSUE #85: Merging PR moves the card
Merging PR moves the card to an appropriate state.

Targetting `master`, `main` or `branch-*` uses `Merge into master` transition.
Targetting any other branch uses `Merge into branch` transition.

### PR DESCRIPTION
**Pull request by pavel-mikula-sonarsource**
Fixes #85

### CODE REVIEW

File: PullRequestClosed/ReadMe.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+jobs:
+  MergePullRequest_job:
+    name: Submit Review
+    runs-on: ubuntu-latest
+    # Single quotes must be used here https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#literals
+    # PRs from forks don't have required token authorization
+    if: |
+        github.event.pull_request.head.repo.full_name == github.repository
+        && github.event.pull_request.merged
```
Comment (sebastien-marichal) [inline PullRequestClosed/ReadMe.md:44]: We should align other examples to whether to include this if or not.
I did not include it in my examples.

**Review by sebastien-marichal — APPROVED** [review]
LGTM!

File: PullRequestClosed/ReadMe.md (hunk 1 above)
Comment (pavel-mikula-sonarsource) [reply PullRequestClosed/ReadMe.md:44]: That's a good point. Please create an issue for it, so we don't forget about it. While thinking about this, the 

> # PRs from forks don't have required token authorization

might be actually a wrong statement, as we only read PR info now. Previously, we also did some write operations.

Comment (pavel-mikula-sonarsource) [reply PullRequestClosed/ReadMe.md:44]: #107

**Pull request merged**